ci-operator/step-registry/gather/extra/gather-extra-commands.sh

```diff
@@ -16,7 +16,9 @@ function queue() {
   fi
 }
 
-export PATH=$PATH:/tmp/shared
+export HOME=/tmp
+export WORKSPACE=${WORKSPACE:-/tmp}
+export PATH=${PATH}:${WORKSPACE}
```

Comment:
While we're touching this and adding new lines, we should quote (e.g. `export PATH="${PATH}:${WORKSPACE}"`).  And maybe also grow ShellCheck coverage for `ci-operator/step-registry`?

Reply:
>  And maybe also grow ShellCheck coverage for ci-operator/step-registry?

not sure what you mean by that... can provide a little more detail more than happy to extend coverage.

> While we're touching this and adding new lines, we should quote (e.g. export PATH="${PATH}:${WORKSPACE}").

Sure will do.

Reply:
> > And maybe also grow ShellCheck coverage for ci-operator/step-registry?
>
> not sure what you mean by that...

The installer has ShellCheck CI to keep us from making quoting mistakes in openshift/installer.  We should have similar CI for this repo (but I can file a PR for that separately; it's orthogonal to this PR).

Reply:
Huh, seems like we [already][1] [do][2].  [This job][3] ran with d5d4e66776's [unquoted `PATH` value][4] and did not fail.  Hmm... are we running a ShellCheck image without mounting in our openshift/release checkout?

[1]: https://github.com/openshift/release/blob/c8de7fa9792ece66b7e90109d6fd0be87a7f3a68/ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml#L721-L737
[2]: https://github.com/openshift/release/blob/c8de7fa9792ece66b7e90109d6fd0be87a7f3a68/hack/validate-registry-commands.sh#L13
[3]: https://deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/origin-ci-test/pr-logs/pull/openshift_release/9640/pull-ci-openshift-release-master-step-registry-shellcheck/1271130929858875392
[4]: https://github.com/openshift/release/commit/d5d4e66776e856892aff66e6ba3746d1927e1514#diff-651f68335975af2efc16bd938e3f474dR21

Reply:
Spinning off into #9772.